dags/monrecap.py

```diff
@@ -23,9 +23,21 @@
 # NOTE: when upgrading to sqlalchemy 2.0 or higher, we'll need to use the class DeclarativeBase
 MonrecapBase = declarative_base()
 
-with DAG("mon_recap", schedule="@daily", **dag_args) as dag:
+with DAG(
+    "mon_recap",
+    schedule="@daily",
+    params={
+        "drop_baro": Param(
+            title="drop table raw_barometre",
+            type="boolean",
+            default=False,
+            description="Drops the raw_barometre table when new columns are added to the file",
```

Comment:
Sinon on pourrais croire que l'option est plus maline qu'elle ne l'est et qu'elle vérifie la condition elle même :).
D'ailleurs je pense que dans les paramètres existants y a un mélange de langue mais on pourrais très bien avoir la règle que `title` et `description` soit en français plutôt qu'en anglais.
```suggestion
            description="Drops the raw_barometre table. Use it when new columns are added to the file",
```

Reply:
Apres j'aurais mis la meme chose en français :sweat_smile: